src/overlay/template.jsx

```diff
@@ -66,17 +64,7 @@ export function Overlay({
       return;
     }
 
-    if (isIos()) {
```

Comment:
"Click to Continue" doesn't seem to do anything with this removed. It appears to be broken now, whereas normally an end user would see:

![Screenshot 2023-10-24 at 10 14 39 AM](https://github.com/paypal/paypal-common-components/assets/20399044/f2546296-2c65-4560-90c3-7c84c482c69e)

Is there a non-blocking alternative? Maybe an alert in `setTimeout`?

Reply:
@nbierdeman thanks for your testing!

Reply:
Just a side note: See that `isIos` is just a regexp on the user agent matching `/iPhone|iPod|iPad/`. Therefore, I'd be interested in the logic that determines whether our popup opens in a tab vs a pop-up window? do you know where to find it, please? Though it opened in a new tab on Safari in general... at least that's what I heard. Would be great to see the prove in the code...

Reply:
we use `window.open` to open the checkout experience. on a desktop, we have options on how to open that window, we choose to open it like a popup

on mobile, the only option is to open it in a new tab. The code looks the same between the platforms, the web browsers themselves dictate how it opens

Reply:
Shouldn't we show that alert for all "mobile" scenarios then? Not just for iOS?

Reply:
I went ahead and made a couple of videos demonstrating @SancSalix's changes. Focus works for Android. I'm unable to test with Firefox due to not being able to get past cert errors.

iOS:
https://github.com/paypal/paypal-common-components/assets/20399044/2899fe22-ee98-457a-b7af-73bfe19ca79c

Android:
https://github.com/paypal/paypal-common-components/assets/20399044/d7c0a46b-d26b-4bd5-8aca-46343ee303de